src/app/admin/createUser/CreateUserForm.tsx

```diff
@@ -75,15 +90,13 @@ const CreateUserForm: React.FC<{}> = () => {
         const { data, error } = await adminInviteUser(fields, redirectUrl);
 
         if (error) {
-            setSubmitError(Errors.external);
+            setServerError(error.type);
             setSubmitDisabled(false);
             setInvitedUser(null);
-
-            const logId = await logErrorReturnLogId("Error with auth: Invite user", error);
-            throw new DatabaseError("insert", "invite user", logId);
+            throw new DatabaseError("insert", "invite user", error.logId);
```

Comment:
I don't think we catch this error anywhere, do we even need to throw? Displaying an error message is how we deal with these errors right? 

And also, maybe we should display the log ID as part of the server error message, so they can tell us what to look for when something goes wrong?

Reply:
yep agreed with both. have done